python_modules/libraries/dagster-components/dagster_components/lib/pipes_subprocess_script_collection.py

```diff
@@ -63,9 +60,7 @@ def get_schema(cls) -> type[PipesSubprocessScriptCollectionParams]:
     def load(
         cls, params: PipesSubprocessScriptCollectionParams, context: ComponentLoadContext
     ) -> "PipesSubprocessScriptCollection":
-        return cls(
-            dirpath=context.path, specs_by_path=params.resolve(context.templated_value_resolver)
-        )
+        return cls(dirpath=context.path, specs_by_path=params.resolve(context.resolve_context))
```

Comment:
one unfortuante part of the context class name is that `resolve_context` looks like a function name

Reply:
still want this addressed